addons/arsenal/functions/fnc_openBox.sqf

```diff
@@ -18,6 +18,11 @@
  * Public: Yes
 */
 
+// Only run this after FUNC(initBox) has run (easier to check if the settings are initialized)
+if !(EGVAR(common,settingsInitFinished)) exitWith {
+    EGVAR(common,runAtSettingsInitialized) pushBack [FUNC(openBox), _this];
+};
```

Comment:
Is this change still necessary?

Reply:
I'll need to take a good look at it again.

Reply:
My testing suggests it isn't needed anymore.